### ISSUE TITLE
Barcodes needed for Covid RDTs

### ISSUE THREAD

**Issue by mwkhan1983** [onaio/rdt-standard#795]
@ellysa we need at least a barcode each for all the Covid RDTs that are currently listed in the app? Can you please help us provide that? You can upload them here on this ticket.

cc: @vincent-karuri

**Comment by ellyasafaa** [comment]
@mwkhan1983 Yes, sure, I can provide you with that. Will ask Dr. Anu for this, but could you explain more what will you do with the barcode? Because the barcode will store different information for each RDT cassette
